dll/win32/shell32/CCopyToMenu.cpp

```diff
@@ -240,6 +240,9 @@ HRESULT CCopyToMenu::DoCopyToFolder(LPCMINVOKECOMMANDINFO lpici)
     }
 
     CStringW strFileTitle = DoGetFileTitle();
+    if (strFileTitle.IsEmpty())
```

Comment:
I am not convinced that the _title_ of a dialog should dictate whether or not we can copy or move a file.

Reply:
@learn-more , we refer to file title (name without extension) not dialog title... Too fast in your comments guy !

Reply:
Ok so everything is fine.

Reply:
You didn't convince me with your weird arguments, so no, I am still not convinced that the title used for a dialog should be used to check if we can copy a file.